### PR TITLE
[validator] Add IsIbanOptions to validator types

### PR DESCRIPTION
**Pull request by edilson**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [url here](https://github.com/validatorjs/validator.js/blob/master/README.md#validators)
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/validator/index.d.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@
 //                 Matteo Nista <https://github.com/Mattewn99>
 //                 Roman Babiak <https://github.com/Almost-Infinity>
 //                 Daniel Freire <https://github.com/dcfreire>
+//                 Edilson Silva <https://github.com/edilson>
```
Comment (jakebailey) [inline types/validator/index.d.ts:17]: If you're looking for credit, there's no need to add yourself here; the git history serves that function. This is the list of owners for the package which will trigger pings, code reviews, etc; are you wanting to do that or is this unintentional?

Comment (edilson) [reply types/validator/index.d.ts:17]: Sorry @jakebailey it was unintentional, will remove that

**Comment by peterblazejewicz** [comment]
@edilson there is also `ESM` export type convention within `es/lib` directory, so it's expected to have a proper re-export there.
For this reason most frequently used implementation is:
- define method within index.d.ts
- re-export defined method from `lib` and from `es/lib`.
Test file contains rows of ES-type imports added just to validate import works, thx!

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@edilson thanks!

**Comment by edilson** [comment]
Ready to merge

**Pull request merged**